modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notificationsconfiguration-cr-by-using-cli_{context}"]
+= Configuring the NotificationsConfiguration CR by using CLI
+
+You can configure the `NotificationsConfiguration` CR by using CLI.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance.
+
+.Procedure
+
+. Enable notifications for the Argo CD instance in the CLI:
++
+[source,terminal]
+----
+$ oc edit argocd openshift-gitops -n openshift-gitops
```

Comment:
Based on the description list used in the following lines, it seems like these are user-replaced values. Per our [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values) they should be formatted this way:
```suggestion
$ oc edit <argo_CD_instance> -n <namespace>
```

If these are not meant to be replaced by the user and will always be those values, ignore my comment.

Reply:
Fixed.